scripts/mikeymike.coffee

```diff
@@ -0,0 +1,76 @@
+##################################
+#
+# NOBODY READ THIS OR ELSE YOU'LL
+# RUIN THE FUN
+#
+#################################
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+module.exports = (robot) ->
+  robot.hear /s (.*) name?/i, (msg) ->
+    nounInQuestion = msg.match[1]
+    if nounInQuestion is "my"
+      msg.reply "IT DOESN'T MATTER WHAT YOUR NAME IS!"
+    else
+      msg.reply "IT DOESN'T MATTER WHAT #{nounInQuestion} IS!""
```

Comment:
The double double-quote at the end of line 76 just needs to be one double-quote. You can fix it and then `commit --amend` to just update your original commit, or change it, add another commit, and squash those two. Both with require a force push after you do it. (Getting to use all that knowledge I imparted yesterday, yay!)